core/lib/via_btc_client/DEV.md

```diff
@@ -200,4 +200,21 @@ Sender Validation: anyone
  !!! if the contract address and call_data was provided the amount get used as fee and remaining amount get sent to l2 receiver address !!!
  !!! in future we can implement kinda enforcement withdrawal with using l1->l2 message (reference in notion) !!!
  !!! also we should support op_return only for bridging in future of the inscription indexer !!!
+
+
+(7)
+ProtocolConfig
+Votable: No
+Sender Validation: only valid sequencer
+|-------------------------------------------------------------|
+|      Schnorr Signature                                      |
+|      Encoded USER/Admin Public Key                          |
+|      OP_CHECKSIG                                            |
+|      OP_FALSE                                               |
+|      OP_IF                                                  |
+|      OP_PUSHBYTES_32  b"Str('via_inscription_protocol')"    |
+|      OP_PUSHBYTES_32  b"Str('bootloader_hash')"             |
```

Comment:
please remove this new message type 

and add these two to bootstrapping message  like this 


OP_PUSHBYTES_32 b"Str('bootloader_hash')"
OP_PUSHBYTES_32 b"Str('abstract_account_hash')

and update whole PR based on this.

(1)
System Bootstrapping Message (txid should be part of genesis state in verifier network)
Sender : Could be anyone
Votable : No
|-------------------------------------------------------------|
|      Schnorr Signature                                      |
|      Encoded Verifier Public Key                            |
|      OP_CHECKSIG                                            |
|      OP_FALSE                                               |
|      OP_IF                                                  |
|      OP_PUSHBYTES_32  b"Str('via_inscription_protocol')"    |
|      OP_PUSHBYTES_32  b"Str('SystemBootstrappingMessage')"  |
|      OP_PUSHBYTES_32  b"start_block_height"                 |
|      OP_PUSHBYTES_32  b"verifier_1_p2wpkh_address"          |
|      OP_PUSHBYTES_32  b"verifier_2_p2wpkh_address"          |
|      OP_PUSHBYTES_32  b"verifier_3_p2wpkh_address"          |
|      OP_PUSHBYTES_32  b"verifier_4_p2wpkh_address"          |
|      OP_PUSHBYTES_32  b"verifier_5_p2wpkh_address"          |
|      OP_PUSHBYTES_32  b"verifier_6_p2wpkh_address"          |
|      OP_PUSHBYTES_32  b"verifier_7_p2wpkh_address"          |
|      OP_PUSHBYTES_32  b"bridge_p2wpkh_mpc_address"          |
| OP_PUSHBYTES_32 b"Str('bootloader_hash')"|
| OP_PUSHBYTES_32 b"Str('abstract_account_hash') | 
|      OP_ENDIF                                               |
|-------------------------------------------------------------|


in future we need new type of message for protocol upgrade, but for now it's better and solid to post these two data as bootstrapping message.

cause if we want to implement it in single and separate message, we should make it voteable so we can bypass an extra voting by putting them on bootstrapping message .

Reply:
Sounds good!

Reply:
@irnb fixed, please review again